CondFormats/SiPixelObjects/src/FrameConversion.cc

```diff
@@ -8,6 +8,84 @@ using namespace std;
 using namespace edm;
 using namespace sipixelobjects;
 
+FrameConversion::FrameConversion(bool bpix, int side, int layer, int rocIdInDetUnit) {
+  int slopeRow =0;
+  int slopeCol = 0;
+  int  rowOffset = 0;
+  int  colOffset = 0; 
+ 
+  if (bpix ) { // bpix 
+    
+    if (side==-1 && layer!=1) {  // -Z side: 4 non-flipped modules oriented like 'dddd', except Layer 1
+
+      if (rocIdInDetUnit <8) {
```

Comment:
hello @veszpv 
there's plenty of magic numbers in this part of the code
can it be made better ?